test/winter_onboarding_2021/4clojure/alisha_test.clj

```diff
@@ -2,8 +2,10 @@
   (:require [clojure.test :refer [deftest is testing]]
             [winter-onboarding-2021.4clojure.alisha :as solutions]))
 
-(deftest function-list-equal-to-quote-list
+(deftest is-function-list-equal-to-quote-list
```

Comment:
This change is unrelated to this commit.

Reply:
Removed the change from the commit message